### PR TITLE
Updating test bed

### PR DESCRIPTION
**Pull request by sojanpr**
Updating test  bed with BUO sharing  model and removing deprecated
methods

@Sarkar @aaustin

### CODE REVIEW

**Comment by aaustin** [comment]
Mm I love it :+1:

File: Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java [hunk 1]
```diff
@@ -341,11 +287,6 @@ public void onInitFinished(BranchUniversalObject branchUniversalObject, LinkProp
 
     }
 
-    @Override
-    protected void onResume() {
-        super.onResume();
-        branchUniversalObject.registerView();
```
Comment (Sarkar) [inline Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java]: Shouldn't we keep this?

**Comment by Sarkar** [comment]
nice - :+1:

File: Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java (hunk 1 above)
Comment (sojanpr) [inline Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java]: We moved this to a new button click

**Pull request merged**